core/transport/pom.xml

```diff
@@ -72,6 +72,26 @@
       <properties>
         <skip.protoc>false</skip.protoc>
       </properties>
+      <build>
+        <plugins>
+          <!-- Cleans the generated java files, prior to regenerating them from the proto/grpc definitions.  -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-clean-plugin</artifactId>
+            <configuration>
+              <filesets>
+                <fileset>
+                  <directory>src/main/java</directory>
+                  <followSymlinks>false</followSymlinks>
+                  <includes>
+                    <include>**/*.java</include>
```

Comment:
can we limit this to `grpc/**/*.java` and `proto/**/*.java`? Don't want to accidentally delete non-autogenerated files